front/src/common/Map/Buttons/MapButtons.tsx

```diff
@@ -113,10 +118,20 @@ export default function MapButtons({
         />,
         'lg'
       );
+    } else if (compact) {
+      openModal(
+        <StdcmLayersModal
+          initialLayers={layersSettings}
+          onChange={({ newLayers }) => {
```

Comment:
You don't need to destructure the newLayers ;)
```suggestion
          onChange={(newLayers) => {
```

Reply:
Yes, I was using this type for the onChange :   ```onChange: (args: { newLayers: LayersSettings }) => void;```
Changed it :)